docs/tables/{custom_resource_name.group_name}.md

```diff
@@ -0,0 +1,556 @@
+| kubernetes_replicaset                 | Kubernetes replica set ensures that a specified number of pod replicas are running at any given time.                                                            |
+| kubernetes_replication_controller     | A Replication Controller makes sure that a pod or homogeneous set of pods are always up and available. If there are too many pods, it will kill some. If there a |
+|                                       | re too few, the Replication Controller will start more.                                                                                                          |
+| kubernetes_resource_quota             | Kubernetes Resource Quota                                                                                                                                        |
+| kubernetes_role                       | Role contains rules that represent a set of permissions.                                                                                                         |
+| kubernetes_role_binding               | A role binding grants the permissions defined in a role to a user or set of users. It holds a list of subjects (users, groups, or service accounts), and a refer |
+|                                       | ence to the role being granted.                                                                                                                                  |
+| kubernetes_secret                     | Secrets can be used to store sensitive information either as individual properties or coarse-grained entries like entire files or JSON blobs.                    |
+| kubernetes_service                    | A service provides an abstract way to expose an application running on a set of Pods as a network service.                                                       |
+| kubernetes_service_account            | A service account provides an identity for processes that run in a Pod.                                                                                          |
+| kubernetes_stateful_set               | A statefulSet is the workload API object used to manage stateful applications.                                                                                   |
+| storeconfigs.secrets.crossplane.io    | Represents Custom resource storeconfigs.secrets.crossplane.io.                                                                                                   |
++---------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+To get details of a specific custom resource table, inspect it by name:
+
+```sql
+select column_name, data_type from INFORMATION_SCHEMA.COLUMNS where table_name like 'certificates.cert-manager.io';
```

Comment:
Do we need to query the `INFORMATION_SCHEMA.COLUMNS` table, or can we just inspect the `certificates.cert-manager.io` table directly?

Reply:
Please see https://hub.steampipe.io/plugins/turbot/prometheus/tables/%7Bmetric_name%7D for an example

Reply:
In the case of CRDs, the table name need to be provided in double quotes; inspect does not support that, already raised an [issue](https://github.com/turbot/steampipe/issues/2455) in Steampipe.